modules/core/src/v2/coins/dot.ts

```diff
@@ -221,6 +210,27 @@ export class Dot extends BaseCoin {
     return { txHex: signedTxHex };
   }
 
+  /**
+   * Explain/parse transaction
+   * @param params
+   * @param callback
+  */
+  async explainTransaction(params: ExplainTransactionOptions): Promise<accountLib.Dot.TransactionExplanation> {
+    const txHex = params.txPrebuild.txHex;
+    if (!txHex || !params.txPrebuild.validity || !params.txPrebuild.key || !params.txPrebuild.referenceBlock || !params.txPrebuild.version) {
```

Comment:
`!params.txPrebuild.version` is it possible for version to ever be `0`?

Reply:
No it cannot be. Version is always 1 or greater.